src/components/product-card/index.tsx

```diff
@@ -0,0 +1,63 @@
+import * as Icon from '../icon';
+import * as Styles from './styles';
+import { Button } from '../button';
+import { Avatar } from '../avatar';
+import { ProductCardProps } from './types';
+import { InboxMessage } from '../inbox-message';
+import { EllipsisText } from '../ellipsis-text'; 
+
+const ProductCard = ({
+  city,
+  text,
+  onClick,
+  picture,
+  buttonText,
+  isFavorite,
+  margin = 0,
+  avatar = '',
+  inboxMessage,
```

Comment:
In the style of the Card component, you have:
margin:  ${margin}  ?  ${margin}px  ||  0;
maybe you don't need to assign 0 here